config/locales/diaspora/en.yml

```diff
@@ -449,6 +449,11 @@ en:
       person_multiple_aspects_a: "Yes. Go to your contacts page and click on “My contacts”. For each contact you can use the menu on the right to add them to (or remove them from) as many aspects as you want. Or you can add them to a new aspect (or remove them from an aspect) by clicking the aspect selector button on their profile page. Or you can even just move the pointer over their name where you see it in the stream, and a “hover-card” will appear. You can change the aspects they are in right there."
       delete_aspect_q: "How do I delete an aspect?"
       delete_aspect_a: "In your list of aspects on the left side of the main page, point your mouse at the aspect you want to delete. Click the little “Edit” pencil that appears on the right. Click the “Delete this aspect” button in the box that appears."
+    chat:
+      title: "Chat"
+      contacts_page: "contacts page"
+      add_contact_roster_q: "How do I add contacts to my roster ?"
+      add_contact_roster_a: "Go to the %{contacts_page}, select an aspect and click on the chat icon to enable chat for the aspect. %{toggle_privilege} Or when you create a new aspect we will also ask you if you would like to enable chat for that aspect"
```

Comment:
Also missing period at the end. How about "for this aspect", or "make contacts of this aspect in the chat available"? I'd drop the "Or " from the next sentence and say "you can choose to" instead of "we will ask you".